client/boot/index.js

```diff
@@ -340,7 +340,7 @@ function boot() {
 	}
 
 	if ( config.isEnabled( 'network-connection' ) ) {
-		require( 'lib/network-connection' ).init();
+		require( 'lib/network-connection' ).init( reduxStore );
```

Comment:
would it make sense to move this immediately below line 164, where we initiate the reduxStore, so that we have the "offline" data when rendering initial components?
```js
reduxStore = createReduxStore();
```